app/models/service_template.rb

```diff
@@ -463,12 +463,14 @@ def update_from_options(params)
     update_attributes!(params.except(:config_info))
   end
 
-  def provision_workflow(user, options = nil)
-    options ||= {}
-    ra_options = { :target => self, :initiator => options[:initiator] }
+  def provision_workflow(user, dialog_options = nil, request_options = nil)
```

Comment:
@bzwei 
provision_workflow is not a private method its going to be called by the REST API. This was done intentionally in https://github.com/ManageIQ/manageiq/pull/13972 so that the REST API can switch overt to using this. From the Rest API they first create a workflow and then submit the request at at later time. In our case when we order the service we want to create the request and immediately submit the request.